types/shared.d.ts

```diff
@@ -63,6 +63,9 @@ export declare type AppearanceOptions = {
   badgeBorderRadius?: string;
   overlayBorderRadius?: string;
 
+  // Overlay
+  overlayZIndex?: string;
```

Comment:
Can we close on the string vs. number topic before merging this one?

Reply:
closed on this - we are going to expose it as a number